reducers/views/threads.ts

```diff
@@ -47,8 +47,19 @@ export function manuallyUnread(state: ViewsState['threads']['manuallyUnread'] |
     return state;
 }
 
+export function toastStatus(state: ViewsState['threads']['toastStatus'] = false, action: GenericAction) {
```

Comment:
Just curiosity. Any idea why are we using this `ViewsState['threads']['toastStatus']` pattern instead of just `boolean` here?

Reply:
I just used it to be consistent with the above functions, it adds no actual benefit